install/recyclarr-install.sh

```diff
@@ -0,0 +1,62 @@
+#!/usr/bin/env bash
+
+# Copyright (c) 2021-2024 tteck
+# Author: tteck (tteckster)
+# License: MIT
+# https://github.com/tteck/Proxmox/raw/main/LICENSE
+source /dev/stdin <<< "$FUNCTIONS_FILE_PATH"
+
+color
+verb_ip6
+catch_errors
+setting_up_container
+network_check
+update_os
+
+msg_info "Installing Dependencies"
+$STD apt-get install -y curl
+$STD apt-get install -y sudo
+$STD apt-get install -y mc
+$STD apt-get install -y tzdata
+$STD apt-get install -y git
+$STD apt-get install -y tini
```

Comment:
Some of these dependencies might not be needed for the binary that was downloaded. They exist in the Dockerfile and aren't referenced in the manual install, so maybe the binary contains its needed dependencies. I could have tested this without the extra dependencies, but I went the safe route to get something up quickly. It'd be good to probably remove these if possible just so that we don't need to worry about them for the update process, which is an oversight because newer recyclarr builds might need newer dependency versions.